web/help/editor.md

```diff
@@ -0,0 +1,32 @@
+Editor Shortcuts
+================
+
+    | `Shortcut`                   | `Action`                     |
+    |------------------------------|------------------------------|
+    | Ctrl + Enter                 | Run the program              |
+    | Ctrl + Space / Shift + Space | Autocomplete                 |
+    | Ctrl + Up                    | Zoom in                      |
+    | Ctrl + Down                  | Zoom out                     |
+    | Ctrl + A                     | Select all                   |
+    | Ctrl + Home                  | Go to start                  |
+    | Ctrl + End                   | Go to end                    |
+    | Alt + Left                   | Go to start of line          |
+    | Alt + Right                  | Go to end of line            |
+    | Ctrl + D                     | Delete line                  |
+    | Ctrl + Left                  | Go one word left             |
+    | Ctrl + Right                 | Go one word right            |
+    | Ctrl + Backspace             | Delete previous word         |
+    | Ctrl + Delete                | Delete next word             |
+    | Ctrl + F                     | Search                       |
+    | Ctrl + G                     | Find next occurrence         |
+    | Ctrl + Shift + G             | Find previous occurrence     |
+    | Ctrl + Shift + F             | Replace                      |
+    | Ctrl + Shift + R             | Replace all                  |
+    | Ctrl + S                     | Save                         |
+    | Ctrl + Z                     | Undo                         |
+    | Ctrl + Shift + Z / Ctrl + Y  | Redo                         |
+    | Ctrl + U                     | Undo selection               |
+    | Ctrl + Shift +  U / Alt + U  | Redo selection               |
+    | Tab / Ctrl + ]               | Indent                       |
+    | Shift + Tab / Ctrl + [       | Unindent                     |
+    | Ctrl + I                     | Reformat (Haskell mode only) |
```

Comment:
Since the keys are different between CodeWorld and Haskell modes, I think we actually want different documents for each mode.  So you'll need a `haskell.shelf` that will initially just contain the API docs and the editor help.  The editor help pages will be almost the same, but autocomplete is only in CodeWorld mode, and Reformat is only in Haskell mode.

Reply:
Done.